src/formatter/json_formatter.ts

```diff
@@ -317,20 +318,33 @@ export default class JsonFormatter extends Formatter {
     pickle: IPickle
     gherkinScenarioMap: { [id: string]: IScenario }
   }): IJsonTag[] {
-    return _.map(pickle.tags, (tagData) => {
-      const featureSource = feature.tags.find(
-        (t: ITag) => t.id === tagData.astNodeId
-      )
-      const scenarioSource = gherkinScenarioMap[pickle.astNodeIds[0]].tags.find(
-        (t: ITag) => t.id === tagData.astNodeId
-      )
-      const line = doesHaveValue(featureSource)
-        ? featureSource.location.line
-        : scenarioSource.location.line
-      return {
-        name: tagData.name,
-        line,
-      }
-    })
+    const scenario = gherkinScenarioMap[pickle.astNodeIds[0]]
+
+    const asJsonTag = (tagData: IPickleTag): IJsonTag =>
```

Comment:
Consider inlining this function in the `map` for better readability

Reply:
Actually I did it that way for readability
But maybe it is unsuccessful 😅

Reply:
Most of the time array fns like `map` and `filter` tend to be inline (at least in JS land). Subjective though, not super fussed about it.